pkg/apis/serving/v1alpha1/revision_validation_test.go

```diff
@@ -690,9 +692,9 @@ func TestImmutableFields(t *testing.T) {
 		want: &apis.FieldError{
 			Message: "Immutable fields changed (-old +new)",
 			Paths:   []string{"spec"},
-			Details: `{v1alpha1.RevisionSpec}.RevisionSpec.ContainerConcurrency:
```

Comment:
There seem no test that tests CC validation anymore?

Reply:
@vagababov there is a `TestContainerConcurrencyValidation` in `pkg/apis/serving/v1beta1/revision_validation_test.go`. I removed ContainerConcurrency from `TestImmutableFields` because the `want` has values for `{v1alpha1.RevisionSpec}.RevisionSpec.ContainerConcurrency` which is a pointer now